v0/webhook/transform.js

```diff
@@ -0,0 +1,35 @@
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  isURL
+} = require("../util");
+
+function process(event) {
+  const { message, destination } = event;
+  const response = defaultRequestConfig();
+  if (destination.Config.webhookUrl && isURL(destination.Config.webhookUrl)) {
+    response.endpoint = destination.Config.webhookUrl;
+    response.method = defaultPostRequestConfig.requestMethod;
+    // add content-type by default as our payload is json
+    response.headers = {
```

Comment:
Our server's network.go support FORM body too. Do we want to restrict webhook only for JSON. 
That is good infact, when someone asks we can enable support for FORM.

Reply:
Makes sense